engine/render/src/render/material.cpp

```diff
@@ -64,7 +62,7 @@ namespace dmRender
 
         if ((constants_count + samplers_count) > 0)
         {
-            m->m_NameHashToLocation.SetCapacity((constants_count + samplers_count) * 2, (constants_count + samplers_count));
+            m->m_NameHashToLocation.SetCapacity((constants_count + samplers_count), (constants_count + samplers_count) * 2);
```

Comment:
I find this in a surprising amount of places.
Our arguments are: table_size (i.e. number of buckets), capacity.
The idea is (in general) to have the table size smaller than the max number of items to be stored. But in many places I see these arguments swapped around.